dags/experiments_live.py

```diff
@@ -47,36 +48,33 @@
         depends_on_past=True,
         parameters=["submission_timestamp:TIMESTAMP:{{ts}}"],
         dag=dag,
+        is_delete_operator_pod=False,
```

Comment:
you need to set this to True everywhere in this dag, and then add the parameter to bigquery_etl_query and gke_command and set the default as False. Then pass them to the GkePodOperators that are returned.

Reply:
done